src/components/forms/SelectBox/SelectBox.js

```diff
@@ -3,28 +3,28 @@ import TextField from '@material-ui/core/TextField';
 import MenuItem from '@material-ui/core/MenuItem';
 import FormControl from '@material-ui/core/FormControl';
 
-import { useForm } from '../../FormProvider/FormProvider';
-
-function SelectBox({ id, formKey, label, options }) {
-  if (formKey == null) formKey = id;
+import { useForm } from 'components/FormProvider';
 
+function SelectBox({ name, label, defaultValue, options }) {
   const { setForm } = useForm();
-  const [selectedItem, setItem] = useState();
+  const [selectedItem, setItem] = useState(defaultValue);
```

Comment:
Same comment re: `defaultValue` as above.

Reply:
I won't repeat myself on the others -- but just pretend like I said this everywhere we have a`defaultValue`. ;-)

Reply:
Same response. ;)